tests/test_physical_systems/test_mechanical_loads.py

```diff
@@ -1,12 +1,31 @@
 import pytest
 import gym_electric_motor as gem
-from gym_electric_motor.physical_systems import PolynomialStaticLoad, MechanicalLoad, ConstantSpeedLoad
+from gym_electric_motor.physical_systems import PolynomialStaticLoad, MechanicalLoad, ConstantSpeedLoad, ExternalSpeedLoad
+from gym.spaces import Box
 import numpy as np
+from scipy import signal
 import math
+from pytest_lambda import lambda_fixture
```

Comment:
Does this package need to be included in the requirements.txt?

Reply:
Yes (required at least for pytest scripts)